static/app/components/events/eventReplay/replayContent.tsx

```diff
@@ -0,0 +1,96 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import DateTime from 'sentry/components/dateTime';
+import Duration from 'sentry/components/duration';
+import Placeholder from 'sentry/components/placeholder';
+import {Provider as ReplayContextProvider} from 'sentry/components/replays/replayContext';
+import ReplayView from 'sentry/components/replays/replayView';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import useFullscreen from 'sentry/utils/replays/hooks/useFullscreen';
+import useReplayData from 'sentry/utils/replays/hooks/useReplayData';
+import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
+
+type Props = {
+  eventSlug: string;
+  orgId: string;
+};
+
+function ReplayContent({eventSlug, orgId}: Props) {
+  const {fetching, replay, fetchError} = useReplayData({
+    eventSlug,
+    orgId,
+  });
+  const {ref: fullscreenRef, toggle: toggleFullscreen} = useFullscreen();
+
+  const replayRecord = replay?.getReplay();
+  const replayEvent = replay?.getEvent();
+
+  if (fetchError) {
+    throw new Error('Failed to load Replay');
+  }
+
+  if (fetching || !replayRecord || !replayEvent) {
+    return <StyledPlaceholder height="400px" width="100%" />;
+  }
+
+  return (
+    <Fragment>
+      <ReplayContextProvider replay={replay} initialTimeOffset={0}>
+        <PlayerContainer ref={fullscreenRef}>
+          <ReplayView toggleFullscreen={toggleFullscreen} />
+        </PlayerContainer>
+      </ReplayContextProvider>
```

Comment:
I assumed we would want to keep it constrained in this instance since it's more of a preview. Full screen gets pretty big on a bigger screen

![Screen Shot 2022-08-08 at 1 28 07 PM](https://user-images.githubusercontent.com/3721977/183477639-a66f85a6-cd63-408c-ba10-f5d6ca5055a9.png)

Reply:
That screen shot looks like full width and full height. I agree that's taking up lots of space, it's too much! I don't mind if the player area is too wide and we get those diagonal lines beside the captured page... how it looks in the end will always depend on the aspect ratio of the capture.